.coveragerc

```diff
@@ -1,6 +1,7 @@
 [run]
-disable_warnings = no-data-collected
```

Comment:
Last but not least, the commit f475704 removes this line. A rationale is provided in its commit message.

> It was important to discover that running the tests with `pytest-xdist` on Travis CI did not produce any coverage reports at all. This setting disabled that warning, which was unfortunate.